ivadomed/scripts/automate_training.py

```diff
@@ -34,7 +40,7 @@ def get_parser():
     parser = argparse.ArgumentParser()
     parser.add_argument("-c", "--config", required=True, help="Base config file path.",
                         metavar=imed_utils.Metavar.file)
-    parser.add_argument("-p", "--params", required=True,
+    parser.add_argument("-ch", "--config-hyper", dest="config_hyper", required=True,
```

Comment:
Could we keep the original flag name?

Reply:
I refactored the name `params` to `config_hyper` because I think it's more descriptive; `params` is a little general. I don't love `--ch` though haha, not sure if there's a better shorthand. `--h` would be better but that is for help...

Reply:
Yeah  I agree `-p` is not great. Okay, let's go with `-ch`. I was just asking for retrocompatibility since I have some files calling this scripts and it is annoying to have to change all of them (mostly in .sh files on ComputeCanada), but I am realizing I don't have that much files to change.

Reply:
Ah I see, maybe in future we should add some retrocompatibility/deprecation warning when changing flags?